test/unittests/execute_call_test.cpp

```diff
@@ -557,19 +557,18 @@ TEST(execute_call, call_nonempty_stack)
     EXPECT_THAT(execute(*instance, 1, {}), Result(3));
 }
 
+constexpr int MaxDepth = 512;
+
 TEST(execute_call, call_imported_infinite_recursion)
 {
     /* wat2wasm
     (import "mod" "foo" (func (result i32)))
-    (func (result i32)
-      call 0
```

Comment:
Why didn't you add it as a new test, but modified this one? This was testing `call` instruction and was in `execute_call` suite, now it goes another code path...

Reply:
Actually I think this code can remain, the point of the change was to assert in the host function. Would just need to check for `MaxDepth - 1` there.

Reply:
Actually this was calling function 0 which is the import. Probably we should duplicate this to have both cases covered.